docs/content/docs/recipes/collaborative-text-area.md

```diff
@@ -24,31 +24,32 @@ This tutorial assumes that you are familiar with the [Fluid Framework Overview](
 
 ## Create the project
 
-1. Open a Command Prompt and navigate to the parent folder where you want to create the project; e.g., `c:\My Fluid Projects`.
+1. Open a Command Prompt and navigate to the parent folder where you want to create the project, e.g., `C:\My Fluid Projects`.
 1. Run the following command at the prompt. (Note that the CLI is np**x**, not npm. It was installed when you installed Node.js.)
 
     ```dotnetcli
-    npx create-react-app collaborative-text-area-tutorial --use-npm
+    npx create-react-app collaborative-text-area-tutorial --template typescript
     ```
 
 1. The project is created in a subfolder named `collaborative-text-area-tutorial`. Navigate to it with the command `cd fluid-react-tutorial`.
 1. The project uses two Fluid libraries:
 
     |Library |Description |
     |---|---|
-    | `fluid-framework`    |Contains the SharedMap and SharedString [distributed data structures]({{< relref "dds.md" >}}) that synchronize data across clients. *This object will hold the most recent timestamp update made by any client.*|
+    | `fluid-framework`    |Contains the SharedString [distributed data structure]({{< relref "dds.md" >}}) that synchronizes text across clients. *This object will hold the most recent text update made by any client.*|
     | `@fluidframework/tinylicious-client`   |Defines the connection to a Fluid server and defines the starting schema for the [Fluid container]({{< relref "containers.md" >}}).|
+    | `@fluid-experimental/react-inputs`   |Contains the SharedStringHelper class that helps provides a simple APIs to interact with the [SharedString]({{< relref "string.md" >}}) object.|
     {.table}
 
     Run the following command to install the libraries.
 
     ```dotnetcli
-    npm install @fluidframework/tinylicious-client fluid-framework
+    npm install @fluidframework/tinylicious-client @fluid-experimental/react-inputs fluid-framework
     ```
 
 ## Code the project
 
-1. Open the file `\src\App.js` in your code editor. Delete all the default `import` statements except the one that imports `App.css`. Then delete all the markup from the `return` statement. The file should look like the following:
+1. Open the file `\src\App.tsx` in your code editor. Delete all the default `import` statements except the one that imports `App.css`. Then delete all the markup from the `return` statement. The file should look like the following:
 
     ```js
```

Comment:
not sure it makes a difference for syntax highlighting/etc but these could be changed to typescript as well